packages/color-field/test/color-field.test.ts

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+
+//import '../sp-color-field.js';
+import '@spectrum-web-components/color-field/sp-color-field.js';
+import { ColorField } from '@spectrum-web-components/color-field';
+import { testForLitDevWarnings } from '../../../test/testing-helpers.js';
+
+describe('ColorField', () => {
+    testForLitDevWarnings(
+        async () =>
+            await fixture<ColorField>(
+                html`
+                    <sp-color-field></sp-color-field>
```

Comment:
We should share fixtures, as they're all pretty similar, or adopt them from stories/a shared [CSF](https://storybook.js.org/docs/api/csf) entry.

Reply:
Done